node/src/components/storage/tests.rs

```diff
@@ -1319,19 +1398,25 @@ impl DatabaseEntriesSnapshot {
 fn should_garbage_collect() {
     let blocks_count = 9_usize;
     let blocks_per_era = 3;
+
+    // Ensure blocks are created with the merkle hashing scheme
+    let merkle_tree_hash_activation = EraId::from(0);
+
     let mut harness = ComponentHarness::default();
-    let mut storage = storage_fixture(&harness);
+    let mut storage = storage_fixture(&harness, merkle_tree_hash_activation);
 
     // Create and store 9 blocks, 0-2 in era 0, 3-5 in era 1, and 6-8 in era 2.
     let blocks: Vec<Block> = (0..blocks_count)
         .map(|height| {
             let is_switch = height % blocks_per_era == blocks_per_era - 1;
-            Block::random_with_specifics(
+
+            Block::random_with_specifics_1(
                 &mut harness.rng,
                 EraId::from((height / blocks_per_era) as u64),
                 height as u64,
                 ProtocolVersion::from_parts(1, 4, 0), // so that they are stored in v2 scheme
```

Comment:
This comment is obsolete.

Reply:
Removed in https://github.com/casper-network/casper-node/pull/2531/commits/15a212035887b1ced40ec3375626d8c251fef585